packages/next-frontend/src/components/create-campaign/form/start-date-step/index.tsx

```diff
@@ -1,26 +1,92 @@
+import { useCallback, useEffect, useState } from "react";
+import { useChainId } from "wagmi";
+import dayjs, { type Dayjs } from "dayjs";
 import { useTranslations } from "next-intl";
 import { Step } from "@/src/components/step";
 import { StepPreview } from "@/src/components/step/preview";
 import { StepContent } from "@/src/components/step/content";
 import type { CampaignPayload, CampaignPayloadPart } from "@/src/types";
+import { Typography } from "@/src/ui/typography";
+import { Button } from "@/src/ui/button";
+import { DateTimePicker } from "@/src/ui/date-time-picker";
+
+import styles from "./styles.module.css";
 
 interface StartDateStepProps {
     disabled?: boolean;
     startDate?: CampaignPayload["startDate"];
-    onStartDateChange?: (startDate: CampaignPayloadPart) => void;
+    endDate?: CampaignPayload["startDate"];
+    onStartDateChange: (startDate: CampaignPayloadPart) => void;
 }
 
 export function StartDateStep({
     disabled,
     startDate,
+    endDate,
     onStartDateChange,
 }: StartDateStepProps) {
     const t = useTranslations("new_campaign.form.start_date");
+    const [open, setOpen] = useState(false);
+    const [date, setDate] = useState<Dayjs | undefined>(startDate);
+    const chainId = useChainId();
+
+    useEffect(() => {
+        setOpen(false);
+    }, [chainId]);
+
+    function handleStepOnClick() {
```

Comment:
useCallback?

Reply:
Fixed in 64b94f25b91cbb89a3677a6ed45bb3df5ea96415.